crypto/blake2b/blake2b_test.go

```diff
@@ -2,32 +2,36 @@ package blake2b
 
 import (
 	"bytes"
-	"github.com/perlin-network/noise/crypto"
 	"math/big"
 	"testing"
+
+	"github.com/perlin-network/noise/crypto"
 )
 
 func TestHash(t *testing.T) {
+	t.Parallel()
 	hp := New()
 
 	r := crypto.Hash(hp, big.NewInt(123))
 
 	n := new(big.Int)
 	n, ok := n.SetString("89391711502145780362310349925943903708999319576398061903082165979787487688967", 10)
-	if ok {
-		if n.String() != r.String() {
-			t.Fatalf("hash error: %v != %v", r, n)
-		}
-	} else {
-		t.Fatal("big int error")
+	if !ok {
+		t.Errorf("big int error")
+	}
+	if n.String() != r.String() {
+		t.Errorf("String() n = %v, want %v", n, r)
 	}
 }
 
 func TestHashBytes(t *testing.T) {
+	t.Parallel()
 	hp := New()
+
 	r := hp.HashBytes([]byte("123"))
+
 	n := []byte{245, 214, 123, 174, 115, 176, 225, 13, 13, 253, 48, 67, 179, 244, 241, 0, 173, 160, 20, 197, 195, 123, 213, 206, 151, 129, 59, 19, 245, 171, 43, 207}
 	if !bytes.Equal(n, r) {
-		t.Fatalf("hash error: %v != %v", r, n)
+		t.Errorf("Equal() n = %v, want %v", n, r)
```

Comment:
Can add testify to go.mod: `github.com/stretchr/testify/assert`. Then if can simplify these tests messages: 
```assert.Equal(t, n, r)```
ie 
https://github.com/perlin-network/perlin-go/blob/master/messages/node_test.go#L119

Reply:
Before approving, we should wait for @jack0 to refactor `network` and `examples`.